### PR TITLE
Bumped Python version of runtime and CI

### PR DESCRIPTION
**Pull request by metamemelord**
This PR introduces Python 3.8 and Python 3.7 into FDK.

### CODE REVIEW

**Review by theatul — APPROVED** [review]
Though i am not familiar with code base enough but version changes seem consistent to me.

File: build-images.sh [hunk 1]
```diff
@@ -18,7 +18,12 @@
 
 set -xe
 
-pyversion=${1:-"3.8.5"}
+if [ -z "$1" ];then
+  echo "Please supply python version to build image. Currently supported versions are 3.6, 3.7, 3.7.1, 3.8, and 3.8.5" >> /dev/stderr
```
Comment (theatul) [inline build-images.sh:22]: Probably better to suggest users to use help command to get supported versions, that will avoid future changes in this message.

**Review by theatul — APPROVED** [review]
Seems fine to me other than one nit.

File: build-images.sh (hunk 1 above)
Comment (metamemelord) [reply build-images.sh:22]: Since this is internal and only used by CircleCI, I'll remove the versions altogether.

**Review by riconnon — APPROVED** [review]
This largely looks fine...
I'm curious about the rationale behind keeping 3.7.1 and 3.8.5 approved nonetheless

**Comment by metamemelord** [comment]
@riconnon this is to maintain backward compatibility. We had discussions with Sachin and we concluded that existing functions should not bump 3.8.5 to 3.8.8 internally. Hence all the functions that use Python 3.7.1 and Python 3.8.5 will continue working but no new apps can be created with these versions.

Also, this is the first phase of FDK work. We plan to send out all the breaking changes in the third phase which includes discontinuation of support for 3.8.5 and 3.7.1.

**Pull request merged**